packages/wrangler/src/init.ts

```diff
@@ -415,6 +415,9 @@ export async function initHandler(args: ArgumentsCamelCase<InitArgs>) {
 					isNamedWorker ? `cd ${args.name} && ` : ""
 				}npm start\``
 			);
+			if (isAddingTests && !packageJsonContent.scripts?.test) {
```

Comment:
Not sure I understand this—is this saying to only print the instruction line if there's no test script?

Reply:
I've extracted these out into their own variable to be clearer, cheers